apps/api/src/app/feature-flags/e2e/get-feature-flag.e2e.ts

```diff
@@ -72,13 +88,26 @@ describe('Get Feature Flag', () => {
         getFeatureFlag = moduleRef.get<GetFeatureFlag>(GetFeatureFlag);
       });
 
-      it.skip(`should get the feature flag value stored in Launch Darkly (enabled)
-         when the SDK key env variable is set regardless of the feature flag set`, async () => {
-        process.env.IS_TEMPLATE_STORE_ENABLED = 'false';
+      describe('IS_TEMPLATE_STORE_ENABLED', () => {
+        it(`should get the feature flag value stored in Launch Darkly (enabled)
+           when the SDK key env variable is set regardless of the feature flag set`, async () => {
+          process.env.IS_TEMPLATE_STORE_ENABLED = 'false';
+
+          const result = await getFeatureFlag.isTemplateStoreEnabled(featureFlagCommand);
 
-        const result = await getFeatureFlag.isTemplateStoreEnabled(featureFlagCommand);
+          expect(result).to.equal(true);
+        });
+      });
 
-        expect(result).to.equal(true);
+      describe('IS_TOPIC_NOTIFICATION_ENABLED', () => {
+        it(`should get the feature flag value stored in Launch Darkly (enabled)
+           when the SDK key env variable is set regardless of the feature flag set`, async () => {
+          process.env.FF_IS_TOPIC_NOTIFICATION_ENABLED = 'false';
+
+          const result = await getFeatureFlag.isTemplateStoreEnabled(featureFlagCommand);
+
+          expect(result).to.equal(true);
+        });
```

Comment:
Testing the E2E to Launch Darkly. We should always test with the environment variable flag set to the opposite value of what's set in Launch Darkly in Test environment.